src/main/java/seedu/address/logic/commands/PersonCommand.java

```diff
@@ -0,0 +1,13 @@
+package seedu.address.logic.commands;
+
+/**
+ * Represents a command with functionality pertaining to persons.
+ */
+public abstract class PersonCommand extends Command {
+    @Override
+    public String getCommandType() {
```

Comment:
Just realized that this may all be redundant if you use the instanceof method, can consider for later iterations